docs/migration.md

```diff
@@ -11,10 +11,10 @@ This document offers guidance on migrating from Prometheus 1.8 to Prometheus 2.0
 
 ## Flags
 
-The format of the Prometheus command line flags has changed. Instead of a
+The format of command line flags has changed. Instead of a
```

Comment:
Let's keep specific about which binary has changed

Reply:
Ah, `promtool`.  I didn't read the original as specific to a binary as written but not worth an argument, reverting.  I felt that "Prometheus" was inferred.